app/src/main/java/org/sopt/diablo/view/ProfileFragment.kt

```diff
@@ -0,0 +1,90 @@
+package org.sopt.diablo.view
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import com.bumptech.glide.Glide
+import org.sopt.diablo.R
+import org.sopt.diablo.data.ServiceCreator
+import org.sopt.diablo.databinding.FragmentProfileBinding
+import org.sopt.diablo.util.MyApplication
+import org.sopt.diablo.util.enqueueUtil
+
+class ProfileFragment: Fragment() {
+    private var position = HomeActivity.FOLLOWER_POSITION
+    private var _binding: FragmentProfileBinding? = null
+    private val binding get() = _binding ?: error("Binding이 초기화 되지 않았습니다")
```

Comment:
이제 이러한 것 보다 코틀린의 위임 속성을 이용하여 보일러 플레이트를 제거하는 것도 좋습니다.
[AutoClearedValue.kt](https://github.com/SOPT-30-ANDROID-1/Harlequin/blob/main/core/src/main/java/happy/mjstudio/core/presentation/util/AuthClearedValue.kt)

```suggestion
    private var binding: FragmentProfileBinding by AutoClearedValue()
```